engine/security/src/main/scala/pl/touk/nussknacker/ui/security/api/AuthenticationConfiguration.scala

```diff
@@ -19,7 +18,7 @@ trait AuthenticationConfiguration {
   def authSeverPublicKey: Option[PublicKey] = Option.empty
   def idTokenNonceVerificationRequired: Boolean
   def implicitGrantEnabled: Boolean
-  def method: AuthenticationMethod
+  def method: String
```

Comment:
Could we use "name" from NamedServiceProvider here? Is there a situation where it can (and should) be different?

Reply:
Renamed